dashboard/static/commands/index.html

```diff
@@ -6,18 +6,18 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Remix - Commands</title>
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
-    <link rel="stylesheet" href="../css/style.css">
+    <link rel="stylesheet" href="../commands/style.css">

```

Comment:
instead of going up 1 directory and going back to commands, you can just do `./style.css`

What you're doing now is basically

```
           static
       /             \
commands     Other folders
/                 \
index.html  style.css
```

instead of going up to static ("../") and then going back down to commands, you can just stay in commands and select style.css there :)

so basically this isn't wrong, but `./style.css` might be less confusing